### DISCUSSION TITLE
Performous 1.2.1

### DISCUSSION THREAD

**Discussion by twollgam** [performous/performous#789] [General]
Right now there are many internal improvements but only a few new features. So we discussed on discord to make a "small" release.

The plan is to create a Performous 1.2.1 before 1.3.0 which should resolve a couple of things:

1. Cleanup os support, automated packages
    - [x] https://github.com/performous/performous/pull/753
    - [x] https://github.com/performous/performous/pull/738
    - [x] https://github.com/performous/performous/pull/732
    - [x] https://github.com/performous/performous/pull/730
2. Bugfixes, refactorings, debugging
    - [x] https://github.com/performous/performous/pull/769
    - [x] https://github.com/performous/performous/pull/782
    - [x] https://github.com/performous/performous/pull/772
    - [x] https://github.com/performous/performous/pull/776
    - [x] https://github.com/performous/performous/pull/755
    - [x] https://github.com/performous/performous/pull/745
3. Completion of translations in terms of string which currently aren't captured for translating and language switching 
    - [x] https://github.com/performous/performous/pull/682
    - [x] https://github.com/performous/performous/pull/747
4. New sorting creteria
    - [x] https://github.com/performous/performous/pull/657
5. Decide on support for Meson build tools
    - [x] Deleted as per: https://github.com/performous/performous/pull/734
6. Fully automated Quality Check / Build / Deploy by using GitHub-Actions and phasing out Semaphore 1 + 2
    - [x] Removal semaphore buildsystem: https://github.com/performous/performous/pull/735
    - [x] Migrate to Github Actions for builds:  https://github.com/performous/performous/pull/715

Basicly the goal is to easy our lifes as developers. but there are only a few new features for the end user (translations, new sorting options)

Thanks for everyone using our new release, if you encounter errors or have any questions feel free to ask them in [user_support on discord (please join)](https://discord.gg/NS3m3ad) or within the Github Issue list 🎉

**Comment by Baklap4** [comment]
Patches should only be used when fixing small things.
I haven't kept track of all the recent changes, but according to the list above a lot has been improved and even some new stuff got added whilst we also delete a lot of things.

For the 1.3.0 backlog we had as goal to ease the lifes of ourselves as developers while maybe introducing a couple of new features. I think that goal is already met.
Another goal was to try close all of the outstanding PR's. This has not yet been achieved, but shouldn't block a release imho.

I'd go for 1.3.0 as it just makes a lot more sense.

Are there currently any major breaking bugs?

**Reply by twollgam** [reply]
If 1.2.1 stands for a bugfix release of 1.2.0 I agree that the next version should be 1.3.0. In my opinion we should not wait for further prs to get ready and merged.

**Comment by Baklap4** [comment]
Gonna lock this discussion so we can continue in https://github.com/performous/performous/discussions/773 and perhaps rephrase what we wanna do
